packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    OwnableUpgradeable
+} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract AttestationStation is Initializable, OwnableUpgradeable, Semver {
+    struct AttestationData {
+        address about;
+        bytes32 key;
+        bytes val;
+    }
+
+    mapping(address => mapping(address => mapping(bytes32 => bytes))) public attestations;
+
+    event AttestationCreated(
+        address indexed creator,
+        address indexed about,
+        bytes32 indexed key,
+        bytes val
+    );
+
+    constructor() Semver(0, 0, 1) {}
```

Comment:
We should call `initialize()` in the `constructor` to prevent the implementation from being uninitialized

Reply:
Should pass `owner` into the constructor and pass to `initialize`

Reply:
removed ownable